scripts/ninja2dot.py

```diff
@@ -1,3 +1,6 @@
+""" This file generates a graph of the dependencies for the ninja build.
+"""
```

Comment:
Hrm.  Is this a recognized indentation?  We usually just put the triple quotes on their own lines.